Sultan_Alsamaani/week_01/day_02/Conditionals_LAB/main.js

```diff
@@ -0,0 +1,44 @@
+var userAge=30;
+if (userAge <18) {
+    console.log("Sorry, you can't drive yet");
+} else if(userAge >=18) {
+    console.log("Drive away!");
+}
+
+var lang = "En";
+if (lang == "En") {
+    console.log("Hello World");
+} else if (lang == "Ar") {
+    console.log("اهلا بالعالم");
+}
+else if (lang == "Fr") {
+    console.log("French French French");
+}
+
+var suburb="sss";
+console.log("i didn`t under stand the request");
+
+var testResult = 90;
+if (testResult >=95) {
+    console.log("A");
+} else if (testResult >=90) {
+    console.log("B");
+} else if (testResult >=80) {
+    console.log("C");
+} else if (testResult >=60) {
+    console.log("D");
+} 
```

Comment:
Nice work here, one thing to think about; what happens if the restResult is less than 60?